.github/workflows/test-coverage.yml

```
@@ -0,0 +1,13 @@
+name: 'coverage'
```

Can we remove the `'` here? and maybe rename it to `Test Coverage`
```suggestion
name: Test Coverage
```

---

✅ 👌 6c62991